page-get-involved.hbs

```diff
@@ -1,132 +1,132 @@
-
-        </div>
-        </a>
-
-        <a href="   https://drive.google.com/file/d/1YvjDW02pklJvOvSDLO9rEaWKk6aZwhPQ/view" target="_blank">
-        <div data-aos="fade-up" class="center"> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster8.png"}}" width="410" height="532">
-
-        </div>
-        </a>
-
-        <a href="   https://drive.google.com/file/d/1nL2pkKEvCxtgNGGDpjd3k13EF0uR5qNP/view" target="_blank">
-        <div data-aos="fade-up" class="right"> <img class="responsive-poster-size hvr-float"  src="{{asset "images/poster9.png"}}" width="410" height="532">
-
-        </div>
-        </a>
-
-    </section>
-    <div  style="display: flex;
-    justify-content: center;margin-bottom: 100px;margin-top: 50px;"> 
-    <a href="https://drive.google.com/file/d/1RmsY8flr9wv4YuBaKHm2dmADoXtUgcJx/view" target="_blank" class="myButton grow2">DOWNLOAD</a>
-    </div>
-
-    {{> footer}}
-
-    {{!-- All the main content gets inserted here, index.hbs, post.hbs, etc --}}
-    {{{body}}}
-
-    <!-- Optional JavaScript -->
-    <!-- jQuery Slim first, then Popper.js, then Bootstrap JS -->
-    <script src="{{asset "js/jquery.min.js"}}"></script>
-    <script src="{{asset "js/popper.min.js"}}"></script>
-    <script src="{{asset "js/bootstrap.min.js"}}"></script>
-    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
-  <script>
-    AOS.init( {once: true}
-    );
-  </script>
-</body>
-
+<!doctype html>
+<html lang="{{@site.lang}}">
+
+<head>
+    <title>{{meta_title}}</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="stylesheet" href="{{asset "css/bootstrap.min.css"}}">
+
+
+    {{!-- Custom Styles --}}
+    <link rel="stylesheet" href="{{asset "css/style.css"}}">
+
+    {{!-- Header CSS --}}
+    <link rel="stylesheet" type="text/css" href="{{asset "css/header.css"}}" />
+
+    {{!-- Footer CSS --}}
+    <link rel="stylesheet" type="text/css" href="{{asset "css/footer.css"}}" />
+
+    {{!-- Hero CSS --}}
+    <link rel="stylesheet" type="text/css" href="{{asset "css/hero.css"}}" />
+
+    {{!-- Animate On Scroll Library --}}
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Roboto&display=swap" rel="stylesheet">
+
+
+
+    {{!-- Dynamic header content for SEO, meta tags, code injection etc --}}
+    {{ghost_head}}
+</head>
+
+<body class="{{body_class}}">
+
+
+    {{> nav}}
+
+        
+    <div id="mission" data-aos="fade-up" class="section-1">
+
+        <div class="text-heading" > <h2> GET INVOLVED </h2></div>
+        <div class="text-dark mx-auto text-left section-width mb-100px"> {{#page}} {{content}} {{/page}} </div>
+        
+    </div>
```

Comment:
Another possible future improvement: what would it take to push as much as possible of this into the user-editable areas of the CMS? It seems unfortunate to have to make PRs to change page titles etc 😞

Reply:
Yeah, definitely need to do a refactor sometime. That would be good change.